spinetoolbox/project_item_icon.py

```diff
@@ -590,6 +590,9 @@ def __init__(self, parent):
     def item_name(self):
         return self._parent.name()
 
+    def _hide_execution_icon(self):
+        self.hide()
```

Comment:
Makes me wonder if we really need this method? Why not just call `hide()`?

Reply:
Thanks, the method is gone now.